ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -152,11 +151,12 @@ const AddCardScreen: React.FC<Props> = props => {
     creditCard.pan
   );
 
-  const updateState = (key: CreditCardStateKeys, value: string) =>
+  const updateState = (key: CreditCardStateKeys, value: string) => {
     setCreditCard({
       ...creditCard,
-      [key]: fromNullable(value)
+      [key]: value.length > 0 ? fromNullable(value) : none
```

Comment:
The same but more functional style
```suggestion
      [key]: fromPredicate((value: string) => value.length > 0)(value)
```

```
import {
  fromPredicate,
} from "fp-ts/lib/Option";
```

Reply:
Thank you :)
Solved in [53a5052](https://github.com/pagopa/io-app/pull/2833/commits/53a5052c828df14d2c6a011406144e0ffd180f4a)